apps/src/clientApi.js

```diff
@@ -1,6 +1,8 @@
 // TODO: The client API should be instantiated with the channel ID, instead of grabbing it from the `dashboard.project` global.
 import queryString from 'query-string';
 
+/*global dashboard*/
```

Comment:
good idea. I wanted to stop using the `window.dashboard` because we've given up checking for `window.dashboard` and it is unnecessarily verbose.